### PR TITLE
Add category model.

### PR DESCRIPTION
**Pull request by Saffarnejad**
Hey behnam please review my code.

### CODE REVIEW

File: SampleWebApplication/Model/Category.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace SampleWebApplication.Model
+{
+    public class Category
+    {
+        public int Id{ get; set; }
```
Comment (bsarraj) [inline SampleWebApplication/Model/Category.cs]: Id propery change to CategoryId.

**Review by bsarraj — COMMENTED** [review]
Please read my comment and refactoring.

File: SampleWebApplication/Model/Category.cs (hunk 1 above)
Comment (vira1368) [reply SampleWebApplication/Model/Category.cs]: I fixed it.

**Comment by Saffarnejad** [comment]
I fixed it.

**Pull request merged**